app/src/ui/branches/pull-request-list-item.tsx

```diff
@@ -51,7 +51,11 @@ export class PullRequestListItem extends React.Component<
   public render() {
     const title = this.props.loading === true ? undefined : this.props.title
     const subtitle = this.getSubtitle()
-
+    const matches = this.props.matches
+    const hasTitleMatches = matches.length > 0
+    const titleMatches = hasTitleMatches ? matches[0] : []
```

Comment:
I'm fine with leaving this as-is rather than refactoring to use `getFirstMatchesOrDefault` because of the `subtitleMatches` bit below.